scripts/fix_ios_embed.sh

```
@@ -0,0 +1,141 @@
+if [ ! -f "$PBXPROJ" ]; then
+    echo "Error: project.pbxproj not found at $PBXPROJ"
+    exit 1
+fi
+
+echo "Fixing iOS project to embed frameworks..."
+
+# Backup original
+cp "$PBXPROJ" "$PBXPROJ.backup"
+
+# Create a temporary file for modifications
+TEMP_FILE=$(mktemp)
```

Shellcheck (0.11.0)

[warning] TEMP_FILE appears unused. Verify use (or export if used externally).

(SC2034)